tests/services/phase.service.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { of } from 'rxjs';
+import { SessionData } from '../../src/app/core/models/session.model';
+import { UserRole } from '../../src/app/core/models/user.model';
+import { Phase, PhaseService } from '../../src/app/core/services/phase.service';
+import { UserService } from '../../src/app/core/services/user.service';
+
+const testStudent = {
+  loginId: 'testStudent',
+  role: UserRole.Student
+};
+
+const testTutor = {
+  loginId: 'testTutor',
+  role: UserRole.Tutor
+};
+
+const mockSettingsFile: {} = {
+  'openPhases' : [Phase.phaseBugReporting],
+  'phaseBugReporting': 'bugreporting',
+  'phaseTeamResponse': 'pe-results',
+  'phaseTesterResponse': 'testerresponse',
+  'phaseModeration': 'pe-evaluation'
+};
```

Comment:
Maybe instead of specifically defining `phaseBugReporting` we could reference the `Phase.phaseBugReporting` as the key to the settings object. This way if we were to change the description of the phases, the test still wouldn't fail? Cause technically we defining the `settings.json` based on our description of the `Phase` enum. But yeah in the `SessionData` model we don't reference the `Phase` enum but I think we eventually should 😅 to standardize things. 

So something along the lines of,
```suggestion
const mockSettingsFile: {} = {
  'openPhases' : [Phase.phaseBugReporting],
  [Phase.phaseBugReporting]: 'bugreporting',
  [Phase.phaseTeamResponse]: 'pe-results',
  [Phase.phaseTesterResponse]: 'testerresponse',
  [Phase.phaseModeration]: 'pe-evaluation'
};
```

What do you think?

Reply:
Yes, I should have actually standardized the convention for the `mockSettingsFile` haha